server/src/main/java/org/opensearch/index/store/remote/utils/cache/LRUCache.java

```diff
@@ -257,12 +258,20 @@ public void decRef(K key) {
 
     @Override
     public long prune() {
+        return prune(null);
```

Comment:
Probably `null` is not a best option but offered an alternative here https://github.com/opensearch-project/OpenSearch/pull/7498/files#r1190262192

Reply:
Thanks! This is better :)